storybook/pages/LoginScreenPage.qml

```diff
@@ -0,0 +1,281 @@
+                    keyUid: "uid_1"
+                },
+                {
+                    keycardCreatedAccount: false,
+                    colorId: 2,
+                    colorHash: [{colorId: 9, segmentLength: 1}, {colorId: 7, segmentLength: 3}, {colorId: 10, segmentLength: 2}],
+                    username: "John",
+                    thumbnailImage: Theme.png("collectibles/CryptoPunks"),
+                    keyUid: "uid_2"
+                },
+                {
+                    keycardCreatedAccount: true,
+                    colorId: 3,
+                    colorHash: [],
+                    username: "8️⃣6️⃣.eth",
+                    thumbnailImage: "",
+                    keyUid: "uid_4"
+                },
+                {
+                    keycardCreatedAccount: true,
+                    colorId: 4,
+                    colorHash: [{colorId: 2, segmentLength: 4}, {colorId: 6, segmentLength: 3}, {colorId: 11, segmentLength: 1}],
+                    username: "Very long username that should eventually elide on the right side",
+                    thumbnailImage: Theme.png("collectibles/SuperRare"),
+                    keyUid: "uid_3"
+                }
+            ]
+            Component.onCompleted: append(data)
+        }
+        onboardingStore: store
+        biometricsAvailable: ctrlBiometrics.checked
+        isBiometricsLogin: localAccountSettings.storeToKeychainValue === Constants.keychain.storedValue.store
+        onBiometricsRequested: biometricsPopup.open()
+        onLoginRequested: (keyUid, method, data) => {
+                              logs.logEvent("onLoginRequested", ["keyUid", "method", "data"], arguments)
+
+                              // SIMULATION: emit an error in case of wrong password
+                              if (method === Onboarding.LoginMethod.Password && data.password !== ctrlPassword.text) {
+                                  onboardingStore.accountLoginError("The impossible has happened", Math.random() < 0.5)
+                              }
+                          }
+        onOnboardingCreateProfileFlowRequested: logs.logEvent("onOnboardingCreateProfileFlowRequested")
+        onOnboardingLoginFlowRequested: logs.logEvent("onOnboardingLoginFlowRequested")
+        onUnlockWithSeedphraseRequested: logs.logEvent("onUnlockWithSeedphraseRequested")
+        onUnlockWithPukRequested: logs.logEvent("onUnlockWithPukRequested")
+        onLostKeycard: logs.logEvent("onLostKeycard")
+
+        // mocks
+        QtObject {
+            id: localAccountSettings
+            readonly property string storeToKeychainValue: ctrlTouchIdUser.checked ? Constants.keychain.storedValue.store : ""
+        }
+        onSelectedProfileKeyIdChanged: biometricsPopup.visible = Qt.binding(() => ctrlBiometrics.checked && ctrlTouchIdUser.checked && store.keycardState === Onboarding.KeycardState.NotEmpty)
+    }
+
+    Dialog {
```

Comment:
It could be excluded to a separate file, could be stored in `pages` dir as some other utilities that are intended to be used only from pages and are not general purpose storybook utils.

Reply:
Yeah good idea; I was thinking about the same for the "keycard state" buttons/combobox, it's just duplicated all over the place

Reply:
Yup, extracted to a separate component

Reply:
putting it into `status-desktop/storybook/src/Storybook/BiometricsPopup.qml`

breaks the convention. `storybook/src/Storybook/` is location of generic storybook components that could be used in any project. App-specific helpers should go directly to `pages` dir.